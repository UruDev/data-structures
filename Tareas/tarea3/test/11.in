# raiz, izquierdo, derecho, buscar_subarbol, linealizacion
insertar_en_binario (12,m)
raiz
izquierdo
derecho
insertar_en_binario (8,g)
insertar_en_binario (9,j)
insertar_en_binario (2,c)
insertar_en_binario (1,a)
insertar_en_binario (14,p)
insertar_en_binario (18,s)
raiz
izquierdo
derecho
insertar_en_binario (4,e)
insertar_en_binario (5,h)
insertar_en_binario (7,k)
insertar_en_binario (1,r)
insertar_en_binario (20,z)
insertar_en_binario (2,n)
raiz
derecho
izquierdo
buscar_subarbol s
buscar_subarbol p
buscar_subarbol t
buscar_subarbol g
buscar_subarbol k
buscar_subarbol c
Fin
