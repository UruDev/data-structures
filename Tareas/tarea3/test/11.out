1>#  raiz, izquierdo, derecho, buscar_subarbol, linealizacion.
2>Insertado.
3>(12,m)
4>
5>
6>Insertado.
7>Insertado.
8>Insertado.
9>Insertado.
10>Insertado.
11>Insertado.
12>(12,m)
13>
-(9,j)
(8,g)
-(2,c)
--(1,a)
14>
-(18,s)
(14,p)
15>Insertado.
16>Insertado.
17>Insertado.
18>Insertado.
19>Insertado.
20>Insertado.
21>(12,m)
22>
--(20,z)
-(18,s)
--(1,r)
(14,p)
-(2,n)
23>
--(7,k)
-(9,j)
--(5,h)
(8,g)
--(4,e)
-(2,c)
--(1,a)
24>(18,s)
25>(14,p)
26>sub es vacÃ­o.
27>(8,g)
28>(7,k)
29>(2,c)
30>Fin.
